### PR TITLE
Readme updates for 1.0-beta.1 release

### PR DESCRIPTION
**Pull request by cholmes**
* Pointed at the published spec instead of just talking about this repo
* Removed implementations and just pointed at website
* Updated 'warning' to not be quite so unstable
* Removed most of roadmap, updated language on it.

### CODE REVIEW

**Comment by cholmes** [comment]
I think ideally we'd move 'goals', 'features' and the roadmap to the website too, and keep the info in the readme more brief, but haven't done that type of stuff on the website yet, so will just leave the content here for now.

File: README.md [hunk 1]
```diff
@@ -54,23 +48,13 @@ A quick overview of what GeoParquet supports (or at least plans to support).
...
-* 0.2 / 0.3 - Feedback from implementations, 3D coordinates support, geometry types, crs optional.
-* 0.x - Several iterations based on feedback from implementations, spatial index best practices.
-* 1.0.0-RC.1 - Aim for this when there are at least 6 implementations that all work interoperably and all feel good about the spec.
-* 1.0.0 - Once there are 12(?) implementations in diverse languages we will lock in for 1.0
-
-Our detailed roadmap is in the [Milestones](https://github.com/opengeospatial/geoparquet/milestones) and we'll aim to keep it up to date.
-
+Our aim is to get to a 1.0.0 in the next few months. The next steps are to push towards our 1.0.0 acceptance criteria, articulated in this [discussion post], which mostly involves getting great validation tools, more software implementations, and more data providers. This should provide key feedback to feel confident in going to 1.0. The goal of 1.0.0 is to establish a baseline of interoperability for geospatial information in Parquet. We've got some exciting plans to go beyond that, which we'll articulate soon on the website.
```
Comment (tschaub) [inline README.md:57]: Looks like a link is missing for the [discussion post].

Maybe:
```suggestion
Our aim is to get to a 1.0.0 in the next few months. The next steps are to push towards our 1.0.0 acceptance criteria, articulated in this [discussion post](https://github.com/opengeospatial/geoparquet/discussions/122), which mostly involves getting great validation tools, more software implementations, and more data providers. This should provide key feedback to feel confident in going to 1.0. The goal of 1.0.0 is to establish a baseline of interoperability for geospatial information in Parquet. We've got some exciting plans to go beyond that, which we'll articulate soon on the website.
```

**Review by tschaub — APPROVED** [review]
Nice improvement.  One suggested change.  If you are making additional changes, it looks like https://github.com/geoarrow/geoarrow is the updated geo-arrow-spec link.

**Review by jorisvandenbossche — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cholmes) [reply README.md]: thanks! missed that.

**Pull request merged**